ccm_web/server/src/canvas/canvas.interfaces.ts

```diff
@@ -93,6 +93,29 @@ export enum UserEnrollmentType {
   Librarian = CanvasRole.Librarian,
   Assistant = CanvasRole.Assistant
 }
+export const clientToCanvasRoleMap: Record<ClientEnrollmentType, UserEnrollmentType> = {
+  student: UserEnrollmentType.StudentEnrollment,
+  teacher: UserEnrollmentType.TeacherEnrollment,
+  ta: UserEnrollmentType.TaEnrollment,
+  observer: UserEnrollmentType.ObserverEnrollment,
+  designer: UserEnrollmentType.DesignerEnrollment,
+  assistant: UserEnrollmentType.Assistant,
+  librarian: UserEnrollmentType.Librarian
+}
+
+export enum ClientEnrollmentType {
+  Student = 'student',
+  Teacher = 'teacher',
+  TA = 'ta',
+  Observer = 'observer',
+  Designer = 'designer',
+  Assistant = 'assistant',
+  Librarian = 'librarian'
+}
```

Comment:
I don't understand your comment. Is comment more informational and suggestion for change? if it is what is being asked as a change?

Reply:
All I'm saying is that you put the definition for `ClientEnrollmentType` after where it is first used in `clientToCanvasRoleMap`. In Python, I believe this would be an error, but JavaScript allows things to be defined later.  I still think it's more readable if the things you use in a file are defined first and then used. Does that make sense? I'm not requiring this.